posts/Steve/2024-11-18-release-announcements.adoc

```diff
@@ -7,10 +7,13 @@ As we have continued automating much of our processes, one item that has become
 That automation creates a https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases[GitHub release] for each release we do.
 
 GitHub exposes an RSS feed for a project's releases.
-For Hibernate ORM, that URL is https://github.com/hibernate/hibernate-orm/releases.atomfootnote:[The format for the RSS feed URL is standardized - https://github.com/{organization}/{project}/releases.atom].
+For Hibernate ORM, that URL is https://github.com/hibernate/hibernate-orm/releases.atom footnote:[The format for the RSS feed URL is standardized - https://github.com/{organization}/{project}/releases.atom].
 
 There are many excellent ways to be notified of releases through this RSS feed, including many email clients.
 And many people already consume these release announcements using RSS from this site.
-For these reasons, we have decided to no longer write blog posts for Hibernate ORM releases.
 
-We will continue to send emails to the hibernate-announce and hibernate-dev mailing lists for some releases.
+For these reasons, we have decided to no longer write blog posts or send emails to the hibernate-announce and hibernate-dev mailing lists
```

Comment:
I suppose we could send (automated) emails for micros, but I don't think we currently have that in our release scripts, so we might as well mention it.